src/main/java/org/opensearch/security/dlic/rest/api/ssl/CertificatesActionType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.dlic.rest.api.ssl;
+
+import org.opensearch.action.ActionType;
+
+public class CertificatesActionType extends ActionType<CertificatesNodesResponse> {
+
+    public static final CertificatesActionType INSTANCE = new CertificatesActionType();
+
+    public static final String NAME = "cluster:admin/security/certificates/info";
```

Comment:
Are we planning to make this permission-able other than the REST API permission `certs/info`

Reply:
AFAIK we are not so far. This is internal `OpenSearch` action type.

Reply:
okay. We evaluate permissions to perform transport action using these action-names, hence I wanted to understand if that is what the intention is here. Should we add a test here to confirm that this action is not permissible when a user has this specific permission `cluster:admin/security/certificates/info`?

Reply:
Ahhh permissions check on the REST layer and only after that this action will be executed. or I'm missing something?

Reply:
This one is about permissions evaluation at Transport layer (the regular evaluator). My question is "Can someone with this specific permission assigned to their role, be able to access cert info? OR will it be admin only?"

Reply:
AFAIK it is possible only in case you use transport client which was removed in 2.x.

Reply:
added